src/samples/AgentToAgent/Agent1/appsettings.json

```diff
@@ -1,8 +1,9 @@
 {
-  "AgentApplicationOptions": {
+  "AgentApplication": {
```

Comment:
I see you changing section names here, but not changing it in the loader?

Reply:
It was wrong in the AgentToAgent samples.  That changed long ago.